README.md

```diff
@@ -1,6 +1,12 @@
 # Beaker tasks used with [skt](https://github.com/RH-FMK/skt) runner
 
-
+## How to run tests
+Here is a list of common prerequisites for all beaker tests, test specific dependencies and steps can be found in the README.md within the test directory.
```

Comment:
Could you put full stop in place of comma after "for all beaker tests"? Also put a dash in "test-specific", and write "within each test's directory"? I.e. write this:
> Here is a list of common prerequisites for all beaker tests. Test-specific dependencies and steps can be found in the README.md within each test's directory.